godtools/App/Share/Extensions/UIViewController+DismissPresented.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  UIViewController+DismissPresented.swift
+//  godtools
+//
+//  Created by Levi Eggert on 12/22/21.
+//  Copyright © 2021 Cru. All rights reserved.
+//
+
+import UIKit
+
+extension UIViewController {
+    
+    // Helper ensures completion will always be called.  I believe if animated is false when calling UIKit's UIViewController.dismiss(animated:) the completion closure will never be called.
+    
+    func dismissPresented(animated: Bool, completion: @escaping (() -> Void)) {
```

Comment:
`dismissPresented` won't get called automatically, right?  We'll have to trigger this ourselves?

Reply:
Hey Robert yeah that's right, we'd have to call this extension.

Reply:
Good to know, thanks!